src/containers/pages/DynamicMDA/plans/index.tsx

```diff
@@ -66,6 +66,7 @@ const mapStateToProps = (
 ): DispatchedStateProps => {
   const searchedTitle = getQueryParams(ownProps.location)[QUERY_PARAM_TITLE] as string;
   const plans = MDAPlansArraySelector(state, {
+    interventionTypes: [InterventionType.MDA],
```

Comment:
This one should be `InterventionType.DynamicMDA`

Reply:
ooh yeah